content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -0,0 +1,171 @@
+=================================
+Use discount and loyalty programs
+=================================
+
+The Odoo *Sales*, *eCommerce*, and *Point of Sale* applications allow users to create discount and
+loyalty programs that customers can use for online and in-store shopping. These programs offer more
+varied, public, and time-sensitive pricing options than :doc:`pricelists <prices/pricing>` do.
+
+Configure the settings
+======================
+
+To begin using discount and loyalty programs, navigate to :menuselection:`Sales --> Configuration
+--> Settings`. Under the :guilabel:`Pricing` heading, activate the :guilabel:`Discounts, Loyalty &
+Gift Card` setting.
+
+Configure discount and loyalty programs
+=======================================
+
+To create discount and loyalty programs, go to :menuselection:`Sales --> Products --> Discount &
+Loyalty`. Then, choose one of the template cards, or click :guilabel:`New` to create a new program
+from scratch. Or, if there are already existing programs, select an existing program to edit it.
+
+.. image:: loyalty_discount/price-discount-loyalty.png
+   :align: center
+   :alt: Discount and loyalty program template cards.
+
+.. note::
+   Templates disappear once the first program is created.
```

Comment:
you could add the note re: Templates only being available on a completely blank (data-less) database here - just a thought